parsec/core/mountpoint/manager.py

```diff
@@ -284,6 +297,8 @@ async def mountpoint_manager_factory(
     # Now is a good time to perform some cleanup in the registry
     if os.name == "nt":
         cleanup_parsec_drive_icons()
+    elif _platform == "darwin" and os.path.isdir(base_mountpoint_path):
```

Comment:
would be better to add a `try/except FileNotFoundError` surrounding the for os.listdir in `cleanup_macos_mountpoint_folder` instead